backend/src/agent/persistence/main.go

```diff
@@ -158,6 +163,7 @@ func init() {
 	// k8s.io/client-go/rest/config.go#RESTClientFor
 	flag.Float64Var(&clientQPS, clientQPSFlagName, 5, "The maximum QPS to the master from this client.")
 	flag.IntVar(&clientBurst, clientBurstFlagName, 10, "Maximum burst for throttle from this client.")
+	flag.StringVar(&executionType, executionTypeFlagName, "Workflow", "Custom Resource's name of the backend Orchestration Engine")
```

Comment:
This code uses the `kind` field of the pipeline to find its execution type. Changing to `ArgoWorkflow` would break the code.

We can think of a better way to do this, maybe using annotation. However, I think this is out of scope of this PR.